warehouse/query-core/src/main/java/datawave/query/util/QueryStopwatch.java

```diff
@@ -24,104 +24,139 @@
 import com.google.common.collect.Queues;
 
 /**
- * 
+ *
  */
 public class QueryStopwatch {
     public static final String NEWLINE = "\n", INDENT = "    ";
     protected ArrayDeque<Entry<String,TraceStopwatch>> watches = Queues.newArrayDeque();
-    
+
     /**
      * Creates a new Stopwatch for use but does not start it
-     * 
+     *
      * @param header
      *            the string header
      * @return new Stopwatch
      */
-    private TraceStopwatch newStopwatch(String header) {
+    TraceStopwatch newStopwatch(String header) {
         checkNotNull(header);
-        
+
         TraceStopwatch sw = new TraceStopwatch(header);
-        
-        watches.add(Maps.immutableEntry(header, sw));
-        
+
+        System.out.println("ohhh");
```

Comment:
This System.out probably no longer needed here :-)

Reply:
hahaha thanks

Reply:
my desperate attempt to find out why the debugger wasn't catching a call only to realize I mockito'd myself in the wrong way.